controllers/dogu_controller_test.go

```diff
@@ -784,6 +785,43 @@ func Test_doguReconciler_checkForAdditionalIngressAnnotations(t *testing.T) {
 	})
 }
 
+func Test_doguReconciler_validateSpecName(t *testing.T) {
+	tests := []struct {
+		name         string
+		recorderFunc func(t *testing.T) record.EventRecorder
+		doguResource *k8sv1.Dogu
+		wantSuccess  bool
+		wantResult   *ctrl.Result
+	}{
+		{
+			name: "should fail validation",
+			recorderFunc: func(t *testing.T) record.EventRecorder {
+				recorder := extMocks.NewEventRecorder(t)
+				recorder.EXPECT().Eventf(mock.Anything, "Warning", "NameValidation", "Dogu resource does not follow naming rules: The dogu's simple name (without the namespace) must equal the resource name. Resource name: %s ; Simple name: %s", "example", "invalid-example")
```

Comment:
Hm, I think, creating the mock and wiping the calls is more a hassle than it's worth, because then we'll also have to manage state. Asserting expectations also wouldn't be as easy then.

Reply:
then leave it as is. It's perfectly fine